devices.js

```diff
@@ -1894,7 +1896,7 @@ const devices = [
         model: '046677548285',
         vendor: 'Philips',
         description: 'Hue white E12 with Bluetooth',
-        extend: hue.light_onoff_brightnes,
+        extend: hue.light_onoff_brightness,
```

Comment:
@Koenkk I don't know what happened here: I never touched this line, but I found a test failing and then I found the typo...not the build passes 🤷‍♂️

Reply:
Was a typo from my side, no problem